configure.in

```diff
@@ -52,7 +52,7 @@ WX_MSW_VERSION=$wx_major_version_number$wx_minor_version_number$wx_release_numbe
 
 dnl Autoconf-2.60 changed the meaning of top_builddir variable, so we have
 dnl to use our own replacement that will work with both 2.5x and 2.60+:
-wx_top_builddir="`pwd`"
+wx_top_builddir="`pwd -W 2> /dev/null || pwd`"
```

Comment:
Hmm, what does `-W` pwd option mean?

Reply:
`pwd -W` produced windows style path using forward slashes, i.e. `c:/my/dir`. It was only available in the Windows version (well, in git bash that is). For Linux there was no `-W` option and the command failed, so the output of plain `pwd` was used.
This path format here was required by `ld`. The same went for the custom dir passed to configure, which upset `ar`. But if there's a better way for obtaining it, no objections here.

Reply:
Hmm, interesting, Cygwin bash doesn't have this option. Also pretty weird that MinGW ld and ar don't understand forward slashes. Again, if we have to do this, let's do it, but ideally I'd like to have this in its own commit too, e.g. to be able to revert it easily later if we find a better solution.

Reply:
Well, they did understand fwd slashes, except for the one before the drive letter. They just wouldn't accept paths starting with `/c/` but only `c:/` (mind the colon too).

Yes I'll separate things in different commits in one of the following days.

Reply:
Weird, all MinGW stuff is supposed to understand `/c` on the command line (MinGW transforms it to whatever is needed). I suspect it somehow doesn't parse ld/ar command line correctly and doesn't recognize the path as being a path or something like that.

What exactly do I need to download to get the version you're using? I have a dozen versions of different MinGWs here but so far I couldn't reproduce this behaviour in a couple of them I tried.

Reply:
`git version` returns `git version 2.11.0.windows.1`. I guess it was taken from the regular git download site... try this: https://github.com/git-for-windows/git/releases/tag/v2.11.0.windows.1
It should come with its own bash which knows `pwd -W`.

The compiler is the one that came with Qt, see the instructions for the online installer. I think it was in a "Tools" section, but it should not be difficult to find among the other items since it was the latest (only?) gcc/mingw one. I used Qt 5.11 BTW.